app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml

```diff
@@ -147,4 +146,45 @@
         <see userInput="Include printed card" stepKey="seePrintCardLabelExists"/>
         <!--Might add assert about the message on Review page-->
     </actionGroup>
+
+    <actionGroup name="CartPageGoToProductEditPanel">
+        <arguments>
+            <argument name="product" defaultValue="ValeriaTwoLayerTank"/>
+        </arguments>
+        <click selector="{{CartPageKebabSection.kebabMenu(product.name)}}" stepKey="clickOnKebabMenu"/>
+        <wait time="3" stepKey="wait123"/>
+        <waitForPwaElementVisible selector="{{CartPageKebabSection.editItem}}" stepKey="waitForKebabMenuToExpand"/>
+        <click selector="{{CartPageKebabSection.editItem}}" stepKey="clickOnEditItem"/>
+        <waitForPwaElementVisible selector="{{EditPanelSection.updateCart}}" stepKey="waitForEditPanelToDisplay"/>
+        <wait time="5" stepKey="temporaryWait"/>
+        <seeElement selector="{{EditPanelSection.updateCart}}" stepKey="checkForEditPanelDisplay"/>
+    </actionGroup>
+
+    <actionGroup name="CartPageEditProductConfigurations">
+        <arguments>
+            <argument name="product" defaultValue="ValeriaTwoLayerTank"/>
+        </arguments>
+        <waitForPwaElementVisible selector="{{EditPanelSection.colorByValue(product.color1)}}" stepKey="waitForProducttoDisplay"/>
+        <click selector="{{EditPanelSection.colorByValue(product.color1)}}" stepKey="clickOnColorOption01"/>
```

Comment:
Can this be a false positive? What if the color and size already selected are the first ones on the list?

Reply:
No, its based on name of swatch.  <data key="color1">Rain</data>